src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./Toggle.module.scss";
+import { Col, Row, Switch, Typography } from "antd";
+import classNames from "classnames";
+
+interface ToggleProps {
+  checked?: boolean;
+  onChange?: (i: boolean) => void;
```

Comment:
`onChange` is not assigned a handler. So, it should be a required prop.

Reply:
Same as above

Reply:
handler is required.

on toggle we always want to do something.